File: mssql/jruby/update/mssql_update.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	mssql_update.rb
#
#				Jan/17/2013
#
# ------------------------------------------------------------
include Java
import com.microsoft.sqlserver.jdbc.SQLServerDriver
import java.sql.DriverManager
import java.lang.System
#
require 'date'
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
#
# ------------------------------------------------------------
puts "*** 開始 ***"
#
id_in = ARGV[0]
population_in = ARGV[1].to_i
#
puts id_in,population_in
#
str_connect = "jdbc:sqlserver://host_mssql;databaseName=city;"
#
conn = DriverManager.getConnection(str_connect,"sa", "scott_tiger")

sql_update_proc(conn,id_in,population_in)
sql_read_proc(conn)

conn.close()
#
puts "*** 終了 ***"
#
# ------------------------------------------------------------

File: mssql/jython/update/mssql_update.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	mssql_update.py
#					Jan/17/2013
#
# ----------------------------------------------------------------
#
import sys
import	string
from java.lang import	System
#
import java
from java.sql import DriverManager
#
# ----------------------------------------------------------------
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_rdb_manipulate import display_proc,update_proc
# ----------------------------------------------------------------
System.out.println ("*** start ***")
print ("*** 開始 ***")
id_in = sys.argv[1]
population_in = string.atoi (sys.argv[2])
print ("%s\t%d" % (id_in, population_in))
#
url="jdbc:sqlserver://host_mssql;databaseName=city;"
user = "sa"
password = "scott_tiger"

conn = DriverManager.getConnection (url,user, password)
update_proc	(conn,id_in,population_in)

display_proc (conn)
#
conn.close()
print ("*** 終了 ***")
#
# ----------------------------------------------------------------
